ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/transaction/RawPrivateTransaction.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.core.requesthandler.sendtransaction.transaction;
+
+import java.math.BigInteger;
+import java.util.List;
+
+import org.web3j.crypto.RawTransaction;
+
+/** Modified from web3j RawTransaction as the web3j class doesn't allow using value as an arg */
```

Comment:
Invalid comment, as PrivateTransaction does not have `value`

Reply:
This should be referring to RawPrivateTransaction not RawTransaction.
RawPrivateTransaction does have a value, it inherits that from PrivateTransaction which has the value field.

Reply:
Updated to refer to web3j RawPrivateTransaction

Reply:
removed this class and using web3j version as we decided we don't care about non-zero values for private transactions